go.mod

```diff
@@ -205,6 +205,7 @@ replace (
 	github.com/dop251/goja_nodejs => github.com/dop251/goja_nodejs v0.0.0-20171011081505-adff31b136e6
 	github.com/fsnotify/fsevents => github.com/elastic/fsevents v0.0.0-20181029231046-e1d381a4d270
 	github.com/fsnotify/fsnotify => github.com/adriansr/fsnotify v0.0.0-20180417234312-c9bbe1f46f1d
+	github.com/golang/glog => github.com/andrewkroh/glog v1.0.1-0.20210831205241-7d8b5c89dfc4
```

Comment:
Could you please move your fork to the elastic organization?

Reply:
Done.